app/assets/javascripts/views/story_view.js

```diff
@@ -137,7 +136,7 @@ module.exports = FormView.extend({
         next_story_id = nextStory.id;
       }
     }
-
+    
```

Comment:
Added an empty line with some spaces. 

Please, check out your editor configurations to make sure trim spaces are correctly set. Moreover, you can adjust your editor to load those expected configurations from .editorconfig file. Check [EditorConfig](http://editorconfig.org/) for more details.